stix2elevator/utils.py

```diff
@@ -183,6 +183,7 @@ def clear_1x_markings_map():
 
 def apply_ais_markings(stix2x_instance, stix2x_marking):
     instance_labels = stix2x_instance.get("labels", []) + stix2x_marking.get("labels", [])
+    # if its an identity object
```

Comment:
I don't understand this comment; I don't see how the code here is checking if it is an identity object.

Reply:
Since you are executing this statement:

> stix2x_instance["created_by_ref"] = stix2x_instance["id"]

the stix2_instance must be an Identity object, which btw, is the only thing you add ais labels to.

Reply:
Would it be more accurate for the comment to say `if the instance is the creator of the marking`?

Reply:
How about:

>  if its an identity object (when the instance is the creator of the marking)

Reply:
That would work.